controller.go

```diff
@@ -42,25 +44,27 @@ type K8SEvent struct {
 }
 
 func (c *K8SController) mainloop(item interface{}) {
-	log.Debug().Str("queue_addr", fmt.Sprintf("%p", &c.queue)).Int("queue_len", c.queue.Len()).Msg("mainloop: running from top")
+	log := c.log.With().Str("where", "mainloop").Int("queue_len", c.queue.Len()).Logger()
```

Comment:
Create a local logger here with info that's relevant to the scope of the function.